<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteGamePage</class>
 <widget class="QWidget" name="DeleteGamePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MainButton" name="deleteBtn">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>320</y>
     <width>160</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MainButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MainButton" name="selectAllBtn">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>320</y>
     <width>80</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>全选</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainButton</class>
   <extends>QPushButton</extends>
   <header>mainbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
